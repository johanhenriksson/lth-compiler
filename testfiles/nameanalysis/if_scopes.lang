int main(int a) {
    if (true) {
        int b = a;
        int c = 5;
    } else {
        int b = c; // error - c is not defined here
        int d = a;
    }
}
