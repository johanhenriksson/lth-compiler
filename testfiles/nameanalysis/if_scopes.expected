Error at line 6,17: symbol 'c' is not declared
